### DISCUSSION TITLE
Customised designs and themes

### DISCUSSION THREAD

**Discussion by nbrown02** [nbrown02/FlowViz-Jira#24] [Show and tell]
Whilst there is an initial design provided with FlowViz - that shouldn't stop you wanting to customise it!

Use this thread as a way to show off custom designs/tweaks you've made :)

**Comment by nbrown02** [comment]
Check out this one from [Karl Furgerot](https://www.linkedin.com/in/karl-furgerot/):

![image](https://github.com/nbrown02/FlowViz-Jira/assets/29369962/f254b786-0f97-43df-bf58-cace34173940)

**Reply by kfurgerot** [reply]
Hi,

If someone is interested by the canva, i can share it :)

**Reply by ImEdmond** [reply]
Hello @kfurgerot I would like to giving it a try :-)
